code/debris/debris.cpp

```diff
@@ -480,6 +481,15 @@ object *debris_create(object *source_obj, int model_num, int submodel_num, vec3d
 
 	float radius = submodel_get_radius(db->model_num, db->submodel_num);
 
+	// if its generic, maybe cull it if its too small and far
+	if (!hull_flag) {
+		// Make vaporize debris seen from farther away
```

Comment:
This comment makes no sense without a `Ship::Ship_Flags::Vaporize` check.

Reply:
Oh whoops, yes, I think I meant to remove the comment as well but this would cull vaporize debris anyway if they passed the check above so it should just ask about vaporizing again here..